internal/constants.go

```diff
@@ -29,6 +29,7 @@ const (
 	FentryProgramFunction       = "bpfman.io.fentryprogramcontroller/function"
 	FexitProgramFunction        = "bpfman.io.fexitprogramcontroller/function"
 	BpfProgramOwnerLabel        = "bpfman.io/ownedByProgram"
+	BpfParentProgram            = "bpfman.io/parentProgram"
```

Comment:
I had trouble coming up with a name for that constant, but I needed a way to get all BpfPrograms that were created as a result of one of the BpfApplication's component programs.  Here are some possible comments.  I'll let @msherif1234 do the update so I don't step on his changes.
	// BpfProgramOwnerLabel is the name of the object that owns the BpfProgram
	// object.  In the case of a *Program, it will be the name of the *Program
	// object.  In the case of a BpfApplication, it will be the name of the
	// BpfApplication object.
	BpfProgramOwnerLabel2 = "bpfman.io/ownedByProgram"
	// BpfParentProgram is the name of the current program that caused the
	// creation of the BpfProgram object.  In the case of a *Program, it will be
	// the name of the *Program object.  In the case of a BpfApplication, it
	// will be the name of the component program.
	BpfParentProgram2        = "bpfman.io/parentProgram"